docs/src/components/ComponentDoc/ComponentDocAccessibility.tsx

```diff
@@ -1,11 +1,19 @@
 import * as React from 'react'
 import * as _ from 'lodash'
-import { Flex, Loader, Text, Provider } from '@stardust-ui/react'
+import { Flex, Loader, Text, Provider, Segment, Divider, Header } from '@stardust-ui/react'
+import ComponentExampleTitle from './ComponentExample/ComponentExampleTitle'
+import BehaviorDescription from './BehaviorDescription'
+import { link } from './../../utils/helpers'
```

Comment:
Let's use `Link` from `react-router-dom`:

```suggestion
import { Link } from 'react-router-dom'
```

Reply:
we agreed with Oleksandr that I will let there "link" component as with "Link" navigation to particular section was not working properly... it is bigger topic for stardust...